src/status_im2/contexts/shell/jump_to/constants.cljs

```diff
@@ -52,18 +52,21 @@
 (def ^:const communities-discover 9)
 
 ;; Floating Screens
```

Comment:
not for this pr, but it feels to me like this sort of information should be defined in one place, e.g `navigations.screens` and we have a mechanism to pass this down to the app. 
right now it feels like we have two navigation systems, the main one and the jump to navigation. 
Is this the approach we want to take or would it suit better to try to keep on navigation system?
cc @smohamedjavid, @briansztamfater

Reply:
hi @J-Son89,

`navigation.screens` is parent ns which holds all the screens. If we use/import this namespace in another file like here it will create cyclic dependency.

> Is this the approach we want to take or would it suit better to try to keep on navigation system?

By navigation system you mean RNN?
RNN is not customizable enough to allow us create UI/animations as per design teams requirment.

Reply:
hmm, it is possible to add custom transition animations with the RNN library, as @briansztamfater has done this on other pages. see the onboarding transitions. Although I am not sure whether that is possible here and not necessarily talking about that.

I am more talking about the code structure we are taking here.
We have navigations/screens which is the core place to store information about screens and then we have this shell information about floating screens (among other info)
imo it is worth considering that we have all navigations pieces in one location, or at least some mechanism to define this information in one place and retrieve it in sub-mechanisms of the application code, such as the shell navigation.